key-vault-node-quickstart-v3/README.md

```diff
@@ -0,0 +1,192 @@
+
+Type the commands below to create a secret in Key Vault called **AppSecret** that will store the value **MySecret**:
+
+```azurecli
+az keyvault secret set --vault-name "<YourKeyVaultName>" --name "AppSecret" --value "MySecret"
+```
+
+To view the value contained in the secret as plain text:
+
+```azurecli
+az keyvault secret show --name "AppSecret" --vault-name "<YourKeyVaultName>"
+```
+
+This command shows the secret information including the URI. After completing these steps, you should have a URI to a secret in an Azure Key Vault. Write this information down. You need it in a later step.
+
+## Clone the Repo
+
+Clone the repo in order to make a local copy for you to edit the source by running the following command:
+
+```
+git clone https://github.com/Azure-Samples/key-vault-node-quickstart.git
+```
+
+## Install dependencies
+
+Here we install the dependencies. Run the following commands:
+```
+cd key-vault-node-quickstart-v3
+```
+```
+npm install
+```
+
+This project used 2 node modules:
+
+* [ms-rest-azure](https://www.npmjs.com/package/ms-rest-azure) 
+* [azure-keyvault](https://www.npmjs.com/package/azure-keyvault)
+
+## Publish the web application to Azure
+
+Below are the few steps we need to do
+
+- The 1st step is to create a [Azure App Service](https://azure.microsoft.com/services/app-service/) Plan. You can store multiple web apps in this plan.
+
+    ```
+    az appservice plan create --name myAppServicePlan --resource-group myResourceGroup
+    ```
+- Next we create a web app. In the following example, replace <app_name> with a globally unique app name (valid characters are a-z, 0-9, and -). The runtime is set to NODE|6.9. To see all supported runtimes, run az webapp list-runtimes
+    ```
+    # Bash
+    az webapp create --resource-group myResourceGroup --plan myAppServicePlan --name <app_name> --runtime "NODE|6.9" --deployment-local-git
+    # PowerShell
+    az --% webapp create --resource-group myResourceGroup --plan myAppServicePlan --name <app_name> --runtime "NODE|6.9"
+    ```
+    When the web app has been created, the Azure CLI shows output similar to the following example:
+    ```
+    {
+      "availabilityState": "Normal",
+      "clientAffinityEnabled": true,
+      "clientCertEnabled": false,
+      "cloningInfo": null,
+      "containerSize": 0,
+      "dailyMemoryTimeQuota": 0,
+      "defaultHostName": "<app_name>.azurewebsites.net",
+      "enabled": true,
+      "deploymentLocalGitUrl": "https://<username>@<app_name>.scm.azurewebsites.net/<app_name>.git"
+      < JSON data removed for brevity. >
+    }
+    ```
+    Browse to your newly created web app and you should see a functioning web app. Replace <app_name> with a unique app name.
+
+    ```
+    http://<app name>.azurewebsites.net
```

Comment:
Here you use lower and space and with angle brackets for placeholder.  As mentioned above be consistent.